app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftConfigurationProperties.java

```diff
@@ -29,11 +27,11 @@
 @Validated
 public class OpenShiftConfigurationProperties {
 
-    public String serviceCaCertFile = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt";
-
     private boolean enabled;
 
-    private String openShiftHost = "https://kubernetes.default.svc";
+    private String masterUrlHost = "https://kubernetes.default.svc";
+
+    private final OpenShiftConfig openShiftClientConfig = new OpenShiftConfigBuilder().withMasterUrl(masterUrlHost).build();
```

Comment:
If you do this then you can't change the master URL via bean properties.

Reply:
Is there a need to change the master URL? I agree we want to make it flexible though if it isn't `kubernetes.default.svc` we're not running on k8s/OpenShift right?

Reply:
Anyhow I can make the change. Shouldn't make much difference...